seaworthy/pytest/fixtures.py

```diff
@@ -97,25 +98,27 @@ def clean_container_fixtures(container, name, scope='class'):
     method will be called on the container object before it is passed as an
     argument to the test function.
 
-    Note that this function returns two fixture functions and references must
-    be kept to both in the correct scope.::
-
-        f1, f2 = clean_container_fixtures(PostgreSQLContainer(), 'postgresql')
-
-        class TestPostgresqlContainer
-            @pytest.mark.clean_postgresql
-            def test_clean_container(self, web_container, postgresql):
-                \"""
-                Test something about the container that requires it to have a
-                clean state (e.g. database table creation).
-                \"""
-
-            def test_dirty_container(self, web_container, postgresql):
-                \"""
-                Test something about the container that doesn't require it to
-                have a clean state (e.g. testing something about a dependent
-                container).
-                \"""
+    .. note:: This function returns two fixture functions. It is important to
+        keep references to the returned functions within the scope of the tests
+        that use the fixtures::
+
+            f1, f2 = clean_container_fixtures(
+                PostgreSQLContainer(), 'postgresql')
```

Comment:
Having to split this over two lines is a pain and makes the generated docs look not-so-good...but I'm not sure how else to appease flake8's line-length check.

Reply:
Maybe keep the example outside the `note` block?